scripts/base/frameworks/config/main.zeek

```diff
@@ -147,6 +147,9 @@ function format_value(value: any) : string
 
 function config_option_changed(ID: string, new_value: any, location: string): any &is_used
 	{
+	# Some option updates reflect Zeek-internal activity and shouldn't be logged.
+	if ( location == "<internal>" )
```

Comment:
Heh, I was there too! I settled for `<internal>` because it's a location, not so much a purpose, but this is perfectly fine with me — I'll go for `<skip-config-log>`.